src/python/pants/util/strutil.py

```diff
@@ -127,3 +128,14 @@ def strip_prefix(string: str, prefix: str) -> str:
     return string[len(prefix):]
   else:
     return string
+
+
+def ensure_relative_file_name(path: Path) -> str:
```

Comment:
Should this be in `dirutil.py`? I'm always unclear where to put new utilities like this.

Also, would be good to add a short test for this utility, especially the edge case of the parameter `path=Path('./')`.

Reply:
This should be in `dirutil.py`! This is v similar to `fast_relpath()`, I believe!